src/ontology/clm.Makefile

```diff
@@ -15,6 +15,10 @@ GENE_LIST = LungMAP LungCellAtlas Neocortex
 GENE_TABLES = $(patsubst %, $(TEMPLATESDIR)/%.tsv, $(GENE_LIST))
 GENE_TEMPLATE = $(TEMPLATESDIR)/genes.tsv
 GENE_TEMPLATE_CL = $(TEMPLATESDIR)/genes_cl.tsv
+# Find all the template files in the templates directory
+GO_TEMPLATES := $(wildcard $(TEMPLATESDIR)/*_quick_go_template.tsv)
+# Pattern to generate OWL files from each template
+GO_TEMPLATE_OWL_FILES := $(patsubst $(TEMPLATESDIR)/%_template.tsv,$(COMPONENTSDIR)/%.owl,$(GO_TEMPLATES))
```

Comment:
this should be `$(TMPDIR)/%.owl` instead of `$(COMPONENTSDIR)/%.owl` to be compatible with `$(TMPDIR)/%.owl: $(TEMPLATESDIR)/%_quick_go_template.tsv` goal